front/src/styles/common.css

```diff
@@ -1,3 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,300;1,400;1,600;1,700;1,800&display=swap');
+
+.auth-card {
+  background-color: #F6F8FB;
+  margin: 60px auto;
+  width: 560px;
+  height: 623px;
+}
+
+.auth-card-title{
+  align-items: center;
+  justify-content: center;
+  padding: 30px 0 0;
+  line-height: 150%;
+}
+
+.input-group:first-child{
+  margin-top: 30px;
+}
+
+.input-group {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.input-group > label {
+  margin: 0 0 5px 18px;
+  font-size: 12px;
+  line-height: 125%;
+  color: black;
+}
+
+.input-group > .label-error {
+  color: #F44336;
```

Comment:
please, check if it is actual after theme changes in dev

it seems like it is common rules not only for login

if it is some type of primary color it should be created css variables in variables.css and use it in other places